h/streamer/app.py

```diff
@@ -0,0 +1,67 @@
+import os
```

Comment:
Good to see an `app.py` file, and everything in `h/streamer/`, that's a log more comprehensible